lib/matplotlib/lines.py

```diff
@@ -592,6 +599,59 @@ def get_markevery(self):
         """return the markevery setting"""
         return self._markevery
 
+    def set_downsample(self, downsample):
```

Comment:
Do we need the `get_`/`set`_? Since this is new, can we just use the `@property` decorator?